### DISCUSSION TITLE
You HAVE to have New Architecture enabled to use >v10.1.20?

### DISCUSSION THREAD

**Discussion by RyanTG** [rnmapbox/maps#3531] [General]
The breaking change description for v10.1.21, released on April 28, states that you cannot upgrade to this rnmapbox version (or beyond) unless you have New Architecture enabled (if you're RN 0.74 or greater).

> this and later release is only compatible with 0.74 when using the new-architecture, old arch should be still working with older RN releases.

Is this an accurate interpretation of the changelog message? So anyone not using New Architecture must stay on v10.1.20? If this is not an accurate interpretation, then I think that changelog message should be revised to be clearer.

**Comment by mfazekas** [comment]
Nope old-arch is supported with rn 0.71+, new-arch only on rn0.74

**Reply by RyanTG** [reply]
Ah, thank you. I now understand that sentence. I still think that sentence should be revised, and I'm not sure if this should count as a _breaking_ change.

`this release is only compatible with 0.74 when using the new-architecture` - The wording can easily be read as `this release is only compatible with _both_ 0.74 and new archictecture1`

My suggestion:

`If you are using the New Architecture, for this and later releases you must be using RN 0.74 or greater. If not using the New Architecture, this and later releases will still work with older RN releases.`
